test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -1398,6 +1407,7 @@ public static final class Builder {
         private int rows;
         private int columns;
         private boolean screenCapture;
+        HyperlinkListener hyperlinkListener;
```

Comment:
```suggestion
        private boolean screenCapture;
        private HyperlinkListener hyperlinkListener;
```
Declare it `private` for consistency.